robottelo/vm.py

```diff
@@ -52,25 +58,34 @@ def __init__(
             image_dir=None):
         self.cpu = cpu
         self.ram = ram
-        if distro is None or distro not in BASE_IMAGES:
-            self.distro = BASE_IMAGES[0]
-        else:
-            self.distro = distro
+        self.distro = BASE_IMAGES[-1] if distro is None else distro
```

Comment:
And it will be default. ```BASE_IMAGES[-1]``` means the last item in the list, and the last item (as per this PR) is ```'rhel71'```